svelte.config.js

```diff
@@ -0,0 +1,26 @@
+import adapter from "@sveltejs/adapter-static";
+import preprocess from 'svelte-preprocess';
+
+const config = {
+  // Consult https://github.com/sveltejs/svelte-preprocess
+  // for more information about preprocessors
+  preprocess: preprocess(),
+  kit: {
+
+    files: {
+      assets: 'src/ui/static',
+      lib: 'src/ui/lib',
+      params: 'src/ui/params',
+      routes: 'src/ui/routes',
+      serviceWorker: 'src/ui/service-worker',
+      appTemplate: 'src/ui/app.html'
+    },
+    adapter: adapter({
+      pages: 'build/ui',
+      assets: 'build/ui',
+      fallback: "index.html",
+    }),
+  },
+};
+
+export default config;
```

Comment:
no newline at the end of the file

Reply:
I dont even know how to fix it at this point, prettier is doing it automatically....................

Reply:
I can set a workspace override for it, and it works, but this has only started happing since using prettier for formatting in the repo